x-pack/legacy/plugins/ml/common/license/ml_license.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Observable, Subscription } from 'rxjs';
+import { ILicense, LICENSE_CHECK_STATE } from '../../../../../plugins/licensing/common/types';
+import { PLUGIN_ID } from '../constants/app';
+
+const MINIMUM_LICENSE = 'basic';
+const MINIMUM_FULL_LICENSE = 'platinum';
```

Comment:
This could get exported and then be reused in `plugins/ml/public/application/management/index.ts` where it's now defined another time.